### PR TITLE
Implement tabbed view support for Attendeees

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/66015 

Builds on: https://github.com/moderntribe/tribe-common/pull/227

This PR uses the Tabbed View to show the attendees report.

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $exclude_modules is never used | [Tickets.php:773](https://github.com/moderntribe/event-tickets/blob/1ec586d4289b03da2306d23e1a95ed9476261e4e/src/Tribe/Tickets.php#L773)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
:bangbang: This Pull Request is against a Central ticket for "Next Release". It shouldn't be merged until the Central ticket has been changed to "Current Release" or "CR Maintenance Release".

File: src/Tribe/Tickets.php [hunk 1]
```diff
@@ -770,10 +770,12 @@ public static function get_event_attendees_count( $event_id ) {
 		 *
 		 * @return array
 		 */
-		public static function get_all_event_tickets( $event_id ) {
+		public static function get_all_event_tickets( $event_id, $exclude_modules = array() ) {
```
Comment (barryhughes) [inline src/Tribe/Tickets.php]: What is the new param for/how is it used - and can we add a matching `@param` tag?

File: src/template-tags/tickets.php [hunk 2]
```diff
@@ -20,12 +20,12 @@
 	 *
 	 * @return bool
 	 */
-	function tribe_events_has_tickets( $event = null ) {
+	function tribe_events_has_tickets( $event = null, $exclude_modules = array() ) {
```
Comment (barryhughes) [inline src/template-tags/tickets.php]: Add a matching `@param` tag?

**Comment by barryhughes** [comment]
:mans_shoe:

**Comment by lucatume** [comment]
Thanks sir.

**Pull request merged**